src/lib/PriorityFeeLib.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import {OutputToken, InputToken} from "../base/ReactorStructs.sol";
+import {PriorityOutput, PriorityInput} from "../lib/PriorityOrderLib.sol";
+import {FixedPointMathLib} from "solmate/src/utils/FixedPointMathLib.sol";
+
+/// @notice helpers for handling priority order objects
+library PriorityFeeLib {
+    using FixedPointMathLib for uint256;
+
+    /// @notice we denominate priority fees in terms of milli-bips, or one thousandth of a basis point
```

Comment:
here's some reasoning:
- want to use relative scaling (increase by X bips per wei of priority fee)
- can't use bips as the denominator here because does not give fillers enough points to fill at on the curve (30 bps of price improvement = max bid of 30 wei)
- so we pick something reasonably smaller than bips but not too small where too much is being paid to the sequencer